jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -358,6 +468,9 @@ YUI.add('user-profile-entity', function() {
         });
         destroyButton = this._generateDestroyButton(entity);
       } else {
+        if (isCharm) {
+          this._getMetrics();
```

Comment:
In a previous review we questioned polling for the metrics every time the component is rendered, which could be often. I think this should go in the `componentDidMount` section.